riscv-types/src/lib.rs

```diff
@@ -0,0 +1,138 @@
+use powdr_isa_utils::SingleDataValue;
+use std::fmt;
+
+#[derive(Clone, Copy, PartialEq, Eq, Debug)]
+pub struct Register {
```

Comment:
These are all needed by both `riscv` and `riscv-elf`, so needed to extract in another crate too.